plugins/system/kernel_same_page_merging

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+#%# family=auto
+
+: << EOF
+=head1 NAME
+kernel_same_page_merging - Outputs various metrics about same page merging.
+
+=head1 AUTHOR
+=over 4
+=item * Michael Grote
+=back
+
+=head1 LICENSE
+GPLv3 or later
+
+SPDX-License-Identifier: GPL-3.0-or-later
+
+=head1 MAGIC MARKERS
+=begin comment
+These magic markers are used by munin-node-configure when installing
+munin-node.
+=end comment
+  #%# family=auto
+=cut
+EOF
+#
+
+# Variablen
+ksm_dir_run="/sys/kernel/mm/ksm/run"
+
+function labels {
+cat << EOF
+graph_title kernel same page merging
+graph_category system
+graph_order pages_unshared pages_volatile pages_shared pages_sharing
+pages_shared.info how many shared pages are being used
+pages_sharing.info how many more sites are sharing them i.e. how much saved
+pages_unshared.info how many pages unique but repeatedly checked for merging
+pages_volatile.info how many pages changing too fast to be placed in a tree
+pages_shared.label pages shared
+pages_sharing.label pages sharing
+pages_unshared.label pages unshared
+pages_volatile.label pages volatile
+pages_shared.draw AREASTACK
+pages_sharing.draw AREASTACK
+pages_unshared.draw AREASTACK
+pages_volatile.draw AREASTACK
+EOF
+}
+
+# prüfe ob ksm aktiviert ist
+if [ ! -f "$ksm_dir_run" ]; then
```

Comment:
Please move this test to the `autoconf` check and add `#%# capabilities=autoconf`.